package.json

```diff
@@ -8,6 +8,8 @@
   "scripts": {
     "abp": "cd node_modules/abp-filter-parser && npm install",
     "postinstall": "npm run abp && export PATH='$PATH:$PWD/node_modules/chromedriver/bin'",
+    "update:tldjs": "npm install --tldjs-update-rules",
```

Comment:
I wasn't sure the best way to do this but I added it as a target to the makefile. https://github.com/duckduckgo/chrome-zeroclickinfo/pull/181/files#diff-b67911656ef5d18c4ae36cb6741b7965R8